bin.js

```diff
@@ -2,101 +2,106 @@
                   'please delete' + path.join(config.path, 'secret'))
 
 var manifestFile = path.join(config.path, 'manifest.json')
 
-// special server command
 if (process.argv[2] == 'server') {
 
-  var createSbot = require('./')
-  .use(require('./plugins/master'))
-  .use(require('./plugins/gossip'))
-  .use(require('./plugins/friends'))
-  .use(require('./plugins/replicate'))
-  .use(require('./plugins/blobs'))
-  .use(require('./plugins/invite'))
-  .use(require('./plugins/block'))
-  .use(require('./plugins/local'))
-  .use(require('./plugins/logging'))
-  .use(require('./plugins/private'))
+  // special server command:
+  // import sbot and start the server
 
+  var createSbot = require('./')
+    .use(require('./plugins/master'))
+    .use(require('./plugins/gossip'))
+    .use(require('./plugins/friends'))
+    .use(require('./plugins/replicate'))
+    .use(require('./plugins/blobs'))
+    .use(require('./plugins/invite'))
+    .use(require('./plugins/block'))
+    .use(require('./plugins/local'))
+    .use(require('./plugins/logging'))
+    .use(require('./plugins/private'))
+
+  // start server
   config.keys = keys
   var server = createSbot(config)
-  fs.writeFileSync(manifestFile, JSON.stringify(server.getManifest(), null, 2))
-  return
-}
 
-// read manifest.json
-var manifest
-try {
-  manifest = JSON.parse(fs.readFileSync(manifestFile))
-} catch (err) {
-  throw explain(err,
-    'no manifest file'
-    + '- should be generated first time server is run'
-  )
-}
+  // write RPC manifest to ~/.ssb/manifest.json
+  fs.writeFileSync(manifestFile, JSON.stringify(server.getManifest(), null, 2))
+} else {
+
+  // normal command:
+  // create a client connection to the server
+
+  // read manifest.json
+  var manifest
+  try {
+    manifest = JSON.parse(fs.readFileSync(manifestFile))
+  } catch (err) {
+    throw explain(err,
+      'no manifest file'
+      + '- should be generated first time server is run'
+    )
+  }
 
-// connect
-require('ssb-client')(keys, {manifest: manifest,
-  port: config.port, host: config.host||'localhost',
-  key: keys.id
-}, function (err, rpc) {
-  if(err) {
-    if (/could not connect/.test(err.message)) {
-      console.log('Error: Could not connect to the scuttlebot server.')
-      console.log('Use the "server" command to start it.')
-      process.exit(1)
+  // connect
+  require('ssb-client')(keys, {
+    manifest: manifest,
+    port: config.port,
+    host: config.host||'localhost',
+    key: keys.id
+  }, function (err, rpc) {
+    if(err) {
+      if (/could not connect/.test(err.message)) {
+        console.log('Error: Could not connect to the scuttlebot server.')
+        console.log('Use the "server" command to start it.')
+        process.exit(1)
+      }
+      throw err
     }
-    throw err
-  }
 
-  // add aliases
-  for (var k in cmdAliases) {
-    rpc[k] = rpc[cmdAliases[k]]
-    manifest[k] = manifest[cmdAliases[k]]
-  }
+    // add aliases
```

Comment:
I did it to balance the code-paths, from

```js
if (server command) {
  code path 1
  return
}
code path 2
```

to

```js
if (server command) {
  code path 1
} else {
  code path 2
}
```

which I think is easier to scan, but idk if you change it back